src/networks/calibnet/mod.rs

```diff
@@ -123,26 +121,17 @@ pub static ref HEIGHT_INFOS: [HeightInfo; 21] = [
     HeightInfo {
         height: Height::Shark,
         epoch: 16_800,
-        bundle: Some(ActorBundleInfo {
-            manifest: Cid::try_from("bafy2bzacedbedgynklc4dgpyxippkxmba2mgtw7ecntoneclsvvl4klqwuyyy").unwrap(),
-            url: Url::parse("https://github.com/filecoin-project/builtin-actors/releases/download/v9.0.3/builtin-actors-calibrationnet.car").unwrap()
-        })
+        bundle: Some(Cid::try_from("bafy2bzacedbedgynklc4dgpyxippkxmba2mgtw7ecntoneclsvvl4klqwuyyy").unwrap())
```

Comment:
Do we still need this? I mean, we have this defined here and in `src/mod.rs`

Reply:
We still need to map the cid to the network and height. Moving everything to mod.rs might be too much